prometheus/push/push.go

```diff
@@ -60,18 +60,18 @@ const contentTypeHeader = "Content-Type"
 // Note that all previously pushed metrics with the same job and other grouping
 // labels will be replaced with the metrics pushed by this call. (It uses HTTP
 // method 'PUT' to push to the Pushgateway.)
-func Registry(job string, grouping map[string]string, url string, reg prometheus.Gatherer) error {
-	return push(job, grouping, url, reg, "PUT")
+func FromGatherer(job string, grouping map[string]string, url string, g prometheus.Gatherer) error {
```

Comment:
I'm not sold on these function names.  I don't find it intuitive, also as none of the other client libraries has the concept of a gatherer. The usage of the client library became very abstract with the introduction of the gatherer interface. I'll need to think about a better name though ...